### PR TITLE
Fix missed ignores

### PR DESCRIPTION
**Pull request by goderbauer**
Preparation for https://github.com/dart-lang/sdk/issues/35234.

### CODE REVIEW

File: packages/flutter/lib/src/painting/notched_shapes.dart [hunk 1]
```diff
@@ -151,10 +151,10 @@ class AutomaticNotchedShape extends NotchedShape {
   final ShapeBorder? guest;
 
   @override
-  Path getOuterPath(Rect hostRect, Rect? guestRect) { // ignore: avoid_renaming_method_parameters, the
-    // parameters are renamed over the baseclass because they would clash
-    // with properties of this object, and the use of all four of them in
-    // the code below is really confusing if they have the same names.
+  Path getOuterPath(Rect hostRect, Rect? guestRect) { // ignore: avoid_renaming_method_parameters
```
Comment (justinmc) [inline packages/flutter/lib/src/painting/notched_shapes.dart:154]: I guess this was just a cleanup and you can't actually remove this ignore.

**Review by justinmc — APPROVED** [review]
LGTM 👍

**Review by chunhtai — APPROVED** [review]
LGTM

**Review by a14n — APPROVED** [review]
LGTM

**Pull request merged**